src/Resource.cs

```diff
@@ -550,6 +536,242 @@ public static void Transform(Vessel v, Resource_recipe recipe)
 		static Dictionary<Guid, Vessel_resources> entries;
 	}
 
+	// equalize/vent a vessel
```

Comment:
what do you think about putting this in its own file for readability 🤷‍♂️

Reply:
I added the code in Resource.cs because this class hosts all activity related to the resource (Cache, Info, Recipe), I thought it was the right place to add a class that would control resource balance.

Reply:
it was just a thought, usually better to keep large classes in their own file, not a big deal tbh